### DISCUSSION TITLE
Solution for the white screen error

### DISCUSSION THREAD

**Discussion by LostFinder2411** [visualboyadvance-m/visualboyadvance-m#1193] [Q&A]
Why do I still get the white screen in VBA - M if I use an official rom?

**Comment by rkitover** [comment]
which official rom, don't link it just give an exact name

**Comment by saulfabregwiivc** [comment]
In what platform are you using VBA-M?

I had the exact same issue when trying to use VBA-M Libretro on Wii, but it was fixed time ago in the code.

Also, is your ROM/BIOS corrupted?
